easybuild/easyconfigs/p/Python/Python-2.7.11-CrayGNU-2015.11-XC.eb

```diff
@@ -0,0 +1,75 @@
+# contributed by Luca MArsella and GPPezzi (CSCS)
+
+name = 'Python'
+version = "2.7.11"
+
+homepage = 'http://python.org/'
+description = "Python is a programming language that lets you work more quickly and integrate your systems more effectively."
+
+toolchain = {'name': 'CrayGNU', 'version': '2015.11-XC'}
+toolchainopts = {'dynamic': True, 'pic': True, 'opt': True, 'optarch': True}
+
+numpyversion = '1.10.4'
+scipyversion = '0.16.1'
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+
+patches = ['unixccompiler-Python-2.7.11.patch']
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.8'),
+    ('libreadline', '6.3'),
```

Comment:
we did a version bump to ncurses 6.0 on our local files, but I forgot that the one on EB is 5.9. 

I think it is easier to stick with 5.9 for this PR, what do you say?